packages/slice-machine/src/modules/availableCustomTypes/index.ts

```diff
@@ -161,6 +182,15 @@ export const availableCustomTypesReducer: Reducer<
         [id]: newCustomType,
       };
     }
+
+    case getType(deleteCustomTypeCreator.success): {
+      const id = action.payload.customTypeId;
+
+      delete state[id];
```

Comment:
The redux state is immutable, don't think you can use `delete` on it. I'm actually surprised it works (unless there's a call to `/state` just after).

You can use

```
return {
  ...state,
  id: undefined
}
``